### PR TITLE
fix: ACNA-1895 update env as part of add service

### PR DESCRIPTION
**Pull request by MichaelGoberling**
## Description

Update the user's local environment with new services when running `aio app add service`. 

## Related Issue

Closes https://github.com/adobe/aio-cli-plugin-app/issues/620

## How Has This Been Tested?

Locally linked plugin with new project & `npm run test`

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by MichaelGoberling** [comment]
Tests are passing locally but failing [here in CI](https://github.com/adobe/aio-cli-plugin-app/actions/runs/3585787913/jobs/6034350339#step:7:959). Can't seem to reproduce the failures locally. This may be due to the tests being run on Ubuntu vs. Mac?

@shazron @moritzraho Thoughts?

**Comment by moritzraho** [comment]
I am not sure what it's going ..  
I am not a big fan of having add services extend use, because the two commands both implement run and achieve a distinct purpose. Maybe  let `app add services` call `app use` (https://oclif.io/docs/running_programmatically#calling-commands-directly) ? (or implement a `syncWithConsole` helper used by both commands ?)

But not sure about the error, it's kinda cryptic, in seems to fail on the line that parses the flags,  maybe it's the inheritance (although it's weird that it works on mac) ?

**Comment by MichaelGoberling** [comment]
PR marked as a draft while work to use a shared helper is being completed.

**Comment by MichaelGoberling** [comment]
@moritzraho @shazron 

Moved the console config import functionality to `import.js` and moved all the underlying helper functions to `import-helper.js`. Now any command should be able to import the latest console config using `import.js`. 

FYI I added minimal export tests to `import.test.js` seeing as `use.test.js` does a pretty good job of testing that functionality already using the different permutations of command flags. Left a comment at the bottom of the test file indicating this.

File: src/lib/import-helper.js [hunk 1]
```diff
@@ -0,0 +1,631 @@
+/*
```
Comment (moritzraho) [inline src/lib/import-helper.js:1]: just to be 100% sure this is just a rename from the old import ? or is it modified too ?

**Review by moritzraho — APPROVED** [review]
Great work ! just one double check comment, otherwise lgtm !

File: src/lib/import-helper.js (hunk 1 above)
Comment (MichaelGoberling) [reply src/lib/import-helper.js:1]: It's just a rename, no real changes were made. A lot of the functions in here are only used by `import.js`, hence the naming :thumbsup:

**Review by shazron — APPROVED** [review]

File: src/lib/import-helper.js (hunk 1 above)
Comment (purplecabbage) [reply src/lib/import-helper.js:1]: Not sure why we are just moving code around like this ... what is the benefit here?

Comment (MichaelGoberling) [reply src/lib/import-helper.js:1]: Yeah it is just moving code but this makes to easier to find exactly what you need to import the console config

The other file contains a lot of file processing funcs which really are just helpers

**Pull request merged**